odl/diagnostics/examples.py

```diff
@@ -21,169 +21,14 @@
 from __future__ import print_function, division, absolute_import
```

Comment:
Absolutely no idea at all why this is in here, I was sure master already had these changes?